ci/slurm_wrapper.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#
+# Executes the given command in the SLURM cluster placing the files in a dynamic location based on the job id last
+# three digits.
```

Comment:
will be nice to include an example of where the files will be located for a job with id 12345